spec/features/users/password_recovery_via_recovery_code_spec.rb

```diff
@@ -2,12 +2,13 @@
 
 feature 'Password recovery via personal key' do
   include PersonalKeyHelper
+  include IdvHelper
 
   let(:user) { create(:user, :signed_up) }
   let(:new_password) { 'some really awesome new password' }
   let(:pii) { { ssn: '666-66-1234', dob: '1920-01-01', first_name: 'alice' } }
 
-  scenario 'resets password and reactivates profile with personal key', email: true, js: true do
+  scenario 'resets password and reactivates profile with personal key', email: true do
```

Comment:
Why remove the JS from this test? Don't we want to have a JS test for the scenario where the reactivation is done with a personal key, as opposed to without a personal key?